features/search/events.feature

```diff
@@ -1,12 +1,25 @@
 Feature:
-
-  Scenario: The search city is base on user's city
+  Background:
     Given the following fixtures are loaded:
       | LoadAdherentData |
+      | LoadEventData    |
+
+  Scenario: The search city is base on user's city
 
     When I am on "/evenements"
     Then the "search-city" field should contain "Paris"
 
     When I am logged as "benjyd@aol.com"
     And I am on "/evenements"
     Then the "search-city" field should contain "Marseille 3e, France"
+
+  Scenario: The display count participant need logging
```

Comment:
As a non logged-in user, I can not see the participants count of an event

Reply:
fixed